pytorch_lightning/profiler/__init__.py

```diff
@@ -197,10 +197,18 @@ def custom_processing_step(self, data):
 from pytorch_lightning.profiler.profilers import (
     AdvancedProfiler,
     BaseProfiler,
+    LegacyPyTorchProfiler,
     PassThroughProfiler,
     PyTorchProfiler,
     SimpleProfiler,
 )
+from pytorch_lightning.utilities import _TORCH_GREATER_EQUAL_1_8
+
+if not _TORCH_GREATER_EQUAL_1_8:
```

Comment:
shall it be 1.8 and greater?

Reply:
There is `GREATER_EQUAL` in the variable. Not sure to get you.